multinode-demo/fullnode.sh

```diff
@@ -31,6 +33,25 @@ EOF
   exit 1
 }
 
+rsync_url() { # adds the 'rsync://` prefix to URLs that need it
```

Comment:
nit: put `rsync_url()` back where you found it :)

Reply:
haha I was looking at that.